NEOSPlus/NEOSPlus.csproj

```diff
@@ -14,6 +14,7 @@
     <NeosPath Condition="Exists('$(HOME)/.steam/steam/steamapps/common/NeosVR/')">$(HOME)/.steam/steam/steamapps/common/NeosVR/</NeosPath>
     <NeosPath Condition="Exists('/mnt/LocalDisk/SteamLibrary/steamapps/common/NeosVR/')">/mnt/LocalDisk/SteamLibrary/steamapps/common/NeosVR/</NeosPath>
     <NeosPath Condition="Exists('C:\Neos\app\')">C:\Neos\app\</NeosPath>
+	<NeosPath Condition="Exists('E:\SteamLibrary/steamapps/common/NeosVR/')">E:\SteamLibrary/steamapps/common/NeosVR/</NeosPath>
```

Comment:
Gonna point this out, I think @Frozenreflex will have something to say about hard-coded paths

Reply:
Agreed, don't really like it either, but is there a better way to point it towards my steam library there?

Reply:
Other than the indentation being wrong, this is the correct way to add your Neos path to the csproj. Could you fix the indentation?